src/PowerShell/Microsoft.WinGet.Configuration.Engine/Commands/ConfigurationCommand.cs

```diff
@@ -434,18 +457,19 @@ private async Task InstallDSCv3Package(OpenConfigurationParameters openParams)
                     Install-Module -Name {WinGetClientModule} -Confirm:$False -Force
                 }}
 
-                $installResult = Install-WingetPackage -Id {DSCv3PackageId} -Source msstore
+                $installResult = Install-WingetPackage -Id {productId} -Source msstore
                 if ($installResult.Status -ne 'Ok')
                 {{
-                    Write-Error ""Failed to install DSCv3 package. Status: $($installResult.Status). ExtendedErrorCode: $($installResult.ExtendedErrorCode)."" -ErrorAction Stop
+                    Write-Error ""Failed to install DSCv3 package. Status: $($installResult.Status). ExtendedErrorCode: $($installResult.ExtendedErrorCode).""
                 }}
                 ");
 
             await installDSCv3.InvokeAsync();
 
             if (installDSCv3.HadErrors)
             {
-                this.Write(StreamType.Error, Resources.ConfigurationInstallDscPackageFailed);
+                this.Write(StreamType.Verbose, installDSCv3.GetErrorMessage() ?? "<Unknown error>");
+                this.Write(StreamType.Warning, Resources.ConfigurationInstallDscPackageFailed);

```

Comment:
Is there a reason we change to warning here?

Reply:
`Error` must be an `ErrorRecord` object, not a string.  I can look at creating one, but there isn't an easy "from a string" method to do so.